docs/website/docs/deployment-configurations/cpu-dylib.md

```diff
@@ -116,7 +116,7 @@ iree/tools/iree-run-module \
     --driver=dylib \
     --module_file=mobilenet-dylib.vmfb \
     --entry_function=predict \
-    --function_inputs="1x224x224x3xf32=0"
+    --function_input="1x224x224x3xf32=0"
```

Comment:
https://github.com/google/iree/blob/main/docs/website/docs/deployment-configurations/gpu-vulkan.md#run-using-the-command-line

also used here and in a few other places

Reply:
Can't figure out how to push to this PR so sent a PR to your branch :P

https://github.com/KareemErgawy-TomTom/iree/pull/1

Reply:
Merged! Now all changes are part of this PR. Thank you.